crates/rooch-rpc-api/src/jsonrpc_types/move_types.rs

```diff
@@ -383,6 +384,78 @@ impl From<H256View> for H256 {
     }
 }
 
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct H64View(
+    #[schemars(with = "Hex")]
+    #[serde_as(as = "Readable<Hex, _>")]
+    [u8; 8],
+);
+
+impl From<H64> for H64View {
+    fn from(value: H64) -> Self {
+        H64View(value.0)
+    }
+}
+
+impl From<H64View> for H64 {
+    fn from(value: H64View) -> Self {
+        H64(value.0)
+    }
+}
+
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct H160View(
+    #[schemars(with = "Hex")]
+    #[serde_as(as = "Readable<Hex, _>")]
+    [u8; 20],
+);
+
+impl From<H160> for H160View {
+    fn from(value: H160) -> Self {
+        H160View(value.0)
+    }
+}
+
+impl From<H160View> for H160 {
+    fn from(value: H160View) -> Self {
+        H160(value.0)
+    }
+}
+
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
+pub struct U256View([u64; 4]);
+
+impl From<U256> for U256View {
+    fn from(value: U256) -> Self {
+        U256View(value.0)
+    }
+}
+
+impl From<U256View> for U256 {
+    fn from(value: U256View) -> Self {
+        U256(value.0)
+    }
+}
+
+#[serde_as]
```

Comment:
We use `StrView<u64>` in json for `u64`. Do we need a `U64View` for `U64` or also use `StrView<u64>`?

Reply:
I changed it to `StrView<u64>`.